app/models/marc_fields/organization_and_arrangement.rb

```diff
@@ -0,0 +1,22 @@
+##
+# MarcField class to parse MARC 351s. The MARC spec indicates that the only
+# human readable subfields for this particular field are $3, $a, $b, and $c.
+class OrganizationAndArrangement < MarcField
+  private
+
+  def preprocessors
+    super + [:whitelist_subfields]
+  end
+
+  def whitelist_subfields
+    relevant_fields.each do |field|
+      field.subfields = field.subfields.select do |subfield|
+        %w(3 a b c).include?(subfield.code)
+      end
+    end
+  end
+
+  def tags
+    ['351']
+  end
```

Comment:
As discussed w/ @ndushay in person, the `tags` method is the API set by the base `MarcField` class so that has farther reaching consequences.